test/e2e/tests/dapp-interactions/dapp-interactions.spec.js

```diff
@@ -24,6 +24,7 @@ describe('Dapp interactions', function () {
       async ({ driver }) => {
         await driver.navigate();
         await openDapp(driver);
+        await driver.delay(1000);
```

Comment:
Why is the delay needed here? Typically we avoid using delays, because they slow the test suite down and can hide flakiness

Reply:
For some reason the button is present and enabled clickable but on clicking it on random occasions does nothing and the MM dialog is not opened. Also tried to wait until the page is loaded, element is present, visible and enabled, however, the flakiness persists. Hence added a hardcoded wait.